staging/src/k8s.io/api/events/v1/types.go

```diff
@@ -0,0 +1,118 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
2020

Reply:
Done. Thanks for catching this. I've updated all v1 files that contain Copyright title to the right year.